test/entity/multipleFieldsEntity.js

```diff
@@ -131,5 +136,67 @@ describe('An entity', () => {
             })
         })
 
+        it('should set a field as ID using id module', () => {
```

Comment:
'should set a field as ID using id module'
what is the 'id module'?
should it be 'should set a field as ID and have valid entity'?

Reply:
I agree with you, it was inside the old PR, but module isn't the best message for this.